### DISCUSSION TITLE
[Feature] Remote access support through Remote Desktop Protocol (RDP)

### DISCUSSION THREAD

**Discussion by ohault** [86Box/86Box#2914] [Ideas]
**Is your feature request related to a problem? Please describe.**
Running 86Box from a tablet/smartphone would be very cool.

**Describe the solution you'd like**
As for VirtualBox (see VirtualRDP), 86Box could provide a way to access it remotely via the Remote Desktop Protocol (RDP).
This could be consider by embedding an existing RDP serveur implementation (see below).  

**Additional context**
- https://github.com/FreeRDP/FreeRDP/tree/master/server
- https://github.com/neutrinolabs/xrdp

**Comment by dhrdlicka** [comment]
There already is support for VNC, although it is not compiled by default.

**Comment by OBattler** [comment]
- dhrdlicka: VNC is currently broken, I believe.

**Comment by solitonmedic** [comment]
Is this still being considered? I’d love to have this.
